cves/CVE-2017-5650.yml

```diff
@@ -119,9 +124,11 @@ discovered:
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
   answer:
-  date:
+    Chun Han Hsiao reported this vulnerability where he discovered that a remote user can consume excessive resources on the target system by
+    sending a series of specially crafted HTTP/2 requests to trigger a flaw in the processing of HTTP/2 GOAWAY frames and consume all available processing threads on the target system.
```

Comment:
I have to horizontally scroll to read these lines. Splitting them up would make them easier to read.

Reply:
fixed